### PR TITLE
Blog post on our new workflow

### PR DESCRIPTION
**Pull request by ABizzinotto**

### CODE REVIEW

File: _posts/2020-06-03-project-workflow [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
```
Comment (lubc) [inline _posts/2020-06-03-project-workflow:1]: @ABizzinotto hey, this file should be `.md`

Comment (ABizzinotto) [reply _posts/2020-06-03-project-workflow]: Done

File: _posts/2020-06-03-project-workflow.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+## What if there are issues after deploying to production?
+
+We should not be shipping bugs to production. If after a Pull Request has been merged and the story has been accepted and deployed to production an issue is found, the story **should not be changed to rejected**.
+
+At that point, a **new** user story should be created with the type set to **bug** and the issue found should be described. The team will then proceed to follow the process to fix the bug.
+
+## Pair Programming
+
+We like to use Pair Programming whenever it makes sense as a technique. If two developers work on an issue and open a Pull Request while pair programming, it is considered code reviewed and Ready for QA. Therefore, it should be labeled as such and a comment should be added to the Pull Request explaining it was opened during a pair programming session.
```
Comment (lubc) [inline _posts/2020-06-03-project-workflow.md:81]: You can link Pair Programming to Fiona's blogpost -> https://www.ombulabs.com/blog/learning/pair-programming/how-to-pair-sucessfully.html

```suggestion
We like to use [Pair Programming](https://www.ombulabs.com/blog/learning/pair-programming/how-to-pair-sucessfully.html) whenever it makes sense as a technique. If two developers work on an issue and open a Pull Request while pair programming, it is considered code reviewed and Ready for QA. Therefore, it should be labeled as such and a comment should be added to the Pull Request explaining it was opened during a pair programming session.
```

File: _posts/2020-06-03-project-workflow.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title:  "The OmbuLabs Workflow: working with GitHub and Pivotal Tracker"
+date: 2020-06-11 11:00:00
```
Comment (lubc) [inline _posts/2020-06-03-project-workflow.md:4]: I can publish this on Monday

File: _posts/2020-06-03-project-workflow.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+<!--more-->
+
+## Step 1: Starting a story
+
+When a developer is ready to start working on a story, they find it in the appropriate project in [Pivotal Tracker](https://www.pivotaltracker.com) and click the Start button. This means they are actively working on implementing that feature, fixing that bug or handling that chore.
+
+## Step 2: Open a Pull Request
+
+When the developer has finished working on the story and is satisfied with the implementation, they open a pull request. To ensure the best person to review the pull request is tagged for review, we use the [GitHub Code Owners](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-file-location) feature. However, the developer can tag more people for review at their discretion.
```
[the remark this answers is not in the document]
Comment (ABizzinotto) [reply _posts/2020-06-03-project-workflow.md:24]: @lubc the goal here was to be gender neutral :) So I used they instead of choosing between he and she.

**Comment by ABizzinotto** [comment]
@lubc I'm not sure about adding the image here. If I were to add it as part of the page, it'd either need to be very big or it would be very hard to read. What do you suggest?

**Comment by lubc** [comment]
> @lubc I'm not sure about adding the image here. If I were to add it as part of the page, it'd either need to be very big or it would be very hard to read. What do you suggest?

true, it would be too big. maybe we can put a link to the image (e.g. https://www.fastruby.io/blog/assets/images/github-pivotaltracker-flow.png) and the reader can click and see it full size.

**Comment by ABizzinotto** [comment]
@lubc good idea!

**Pull request merged**